### PR TITLE
Add section about scroll timeline

### PR DESCRIPTION
**Pull request by majido**

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -659,6 +656,18 @@ Timeline Attachment {#timeline-attachment}
 
 Issue(61): Define semantics of attachment and detachment.
 
+ScrollTimeline {#scroll-timeline}
+---------------------------------
+{{ScrollTimeline}} is a new concept being proposed for addition to web animation API. It defines
+an animation timeline whose time value depends on scroll position of a scroll container.
```
Comment (stephenmcgruer) [inline index.bs]: 'the scroll position'

File: index.bs [hunk 2]
```diff
@@ -659,6 +656,18 @@ Timeline Attachment {#timeline-attachment}
...
+<a>Worklet animations</a> can have a scroll timeline as thus drive its scripted effect based on
```
Comment (stephenmcgruer) [inline index.bs]: 'and thus', 'drive their', 'effects'

Comment (majido) [reply index.bs]: Ouch! So many mistakes in one sentence :cry: Fixed.

File: index.bs (hunk 1 above)
Comment (majido) [reply index.bs]: Done.

**Pull request merged**